package foo

func main() {
	log.Error("whoops")
}
